pkg/services/chats/chatmodel/models.go

```diff
@@ -0,0 +1,71 @@
+package chatmodel
+
+const (
+	// ContentTypeOrg is reserved for future use for per-org discussions.
+	ContentTypeOrg = 1
+	// ContentTypeDashboard used for dashboard-wide discussions.
+	ContentTypeDashboard = 2
```

Comment:
the magic numbers are a bit confusing and maybe limited... maybe better would be constant strings.  For example:
* dash
* anno

comments on individual panels seems valuable -- but also nervous about a page with a subscription for each panel.

Reply:
Using strings for content types now, in database table too: NVarChar(10). And strings in API layer.